tests/Feature/Http/Controllers/RestApi/AuthHelper.php

```diff
@@ -54,6 +54,7 @@ private function addClient(array $rootGroups)
     {
         Artisan::call('client:add', [
             'name' => 'Unit Test',
+            'webling-key' => config('app.webling_api_key'),
```

Comment:
Wieder laut gedacht zur Nachvollziehbarkeit: Das ist innerhalb eines Tests und daher funktioniert die Injection nicht.

Reply:
Der Key ist ein zwingendes Argument für `client:add`, ergo muss auch bei den Tests ein Key angegeben werden. Die Überlegung dahinter war, dass der/die User*in die einen neuen Token hinzufügt, sich somit explizit Gedanken zu den Zugriffsrechten machen muss und einen entsprechenden Key im Webling erstellen muss. So vermeiden wir die Gefahr, einfach mal mit dem default Key zu arbeiten und danach zu vergessen, dass dieser womöglich zu viel Rechte hat.